docs/patterns/clustered-listening-tentacles.md

```diff
@@ -0,0 +1,151 @@
+Tentacle.exe import-certificate -instance "Tentacle" -f "TentacleClusterPrivateKey.txt"
+Tentacle.exe service --instance "Tentacle" --stop --start
+```
+
+Now on the second node, stop the Tentacle service and bring the shared storage offline the same way you have done before. This time go back to the first node in the cluster, bringing the Shared Storage drive back online and start the tentacle service. Copy the newly exported "TentacleClusterPrivateKey.txt" file to the first node, then follow the step above to import it into the first node to ensure both nodes in the cluster hold the same private key. 
+
+Once both Tentacles are installed and configured ensure that neither node has the Octopus Tentalce started and that the shared storage is brought offline.
+
+*![](/docs/images/clustered-listening-tentacles/import-privatekey.jpg)*
+
+## Configure A new Clustered Service {#ClusteringTentacles-NewCluster}
+
+Ensure each node that will be participating in the Tentacle Cluster is joined to the Active Directory Domain and has the **Failover Clustering** Feature installed in Windows.
+
+Open the **Failover Cluster Manager** console on one of the nodes. If there is no cluster configured yet, you can right click **Failover Cluster Manager** amd select **New Cluster**
+
+On the **Select Servers** page, enter the Fully Qualified Domain Name of each node that will be in this cluster.
+
+On the **Validation Warning** page, select "Yes, When I click Next, run configuration validation tests, and then return to the process of creating the cluster"
+
+The **Validate a Configuration Wizard** appears, select **Run all Tests** and select **Next**
+
+When all validation appears OK, you will be returned to the **Create Cluster Wizard** where the **Access Point for Administering the Cluster** page appears, Choose an IP Address that is on the same Network as both Tentacles and a hostname that is 15 characters or less.
+
+*![](/docs/images/clustered-listening-tentacles/configure-clusterhostname.jpg)*
+
+Now complete the Wizard.
+
+## Adding Octopus Tentacle as a Generic Service Cluster {#ClusteringTentacles-AddTentacleCluster}
+
+Right-Click **Roles** and select **Configure Roles** then Highlight **Generic Service**, click **Next**.
+
+*![](/docs/images/clustered-listening-tentacles/cluster-newrolewizard-servicetype.jpg)*
+
+Find and highlight **OctopusDeploy Tentacle** service in the list of available services, click **Next**
+
+*![](/docs/images/clustered-listening-tentacles/cluster-newrolewizard-selectservice.jpg)*
+
+Under **Client Access Point** choose an apropriate NetBIOS Name and IP Address for this clustered role. Note down this IP Address/DNS Hostname, you will be using it in the Octopus Server for connection to the Tentacle Cluster.
+
+*![](/docs/images/clustered-listening-tentacles/cluster-newrolewizard-clientaccess.jpg)*
+
+Under **Select Storage** choose the Disk which we had mounted previously and installed the Home and Application folders to previously
```

Comment:
`Disk` doesn't need to be capitalised
`choose the Disk which we had mounted previously and installed the Home and Application folders to previously` =>`choose the disk that is configured as shared storage`